yaas/parsers/host.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+
+from .. import host
+from .. import config
+from .. import utils
+
+def define_subcommand(parent_parser):
+  parser = parent_parser.add_parser(
+      'host',
+      help=inspect.getdoc(host))
+
+  subcommands = [
+      host_ls,
+      host_show,
+      ]
+
+  subparsers = parser.add_subparsers()
+  for subcommand in subcommands:
+      subcommand(subparsers)
+
+def host_ls(parent_parser):
+    def ls(args):
```

Comment:
I explicitly put them inside the parser functions because they shouldn't need to be accessible outside. They should generally be thin wrappers around the yaas module functions. If they have similar logic (e.g. printing) that should go in utils.py.